test/command_runner_test.dart

```diff
@@ -131,7 +179,9 @@ Run "test help <command>" for more information about a command.'''));
     });
 
     test("doesn't print hidden commands", () {
-      runner..addCommand(HiddenCommand())..addCommand(FooCommand());
+      runner
+        ..addCommand(HiddenCommand())
+        ..addCommand(FooCommand());
```

Comment:
probably a good idea to add a hidden, categorized command in here as well?

Maybe also test that the category header is not displayed at all if all its commands are hidden?

Reply:
Good idea, done.